src/test/java/service/FruitParserImplTest.java

```diff
@@ -0,0 +1,44 @@
+package service;
+
+import model.FruitTransaction;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FruitParserImplTest {
+    private static FruitParser fruitParser;
+
+    @BeforeClass
+    public static void beforeClass() {
+        fruitParser = new FruitParserImpl();
+    }
+
+    @Test
+    public void parsedData_Ok() {
+        FruitTransaction expected = new FruitTransaction();
+        expected.setOperation(FruitTransaction.Operation.BALANCE);
+        expected.setFruit("banana");
+        expected.setQuantity(20);
+        String actualData = " b,banana,20";
+        FruitTransaction actual = fruitParser.parseFruitTransaction(actualData);
+        Assert.assertEquals(expected, actual);
+    }
+
+    @Test(expected = RuntimeException.class)
+    public void nullOperationData_NotOk() {
+        String actualData = " ,banana,20";
+        fruitParser.parseFruitTransaction(actualData);
+    }
+
+    @Test(expected = NullPointerException.class)
```

Comment:
we should not expect NullPoinerException, lets add check for this case in your code